learning_wavelets/training_scripts/exact_recon_unet_training.py

```diff
@@ -0,0 +1,95 @@
+import os
+import os.path as op
+import time
+
+from tensorflow.keras.callbacks import TensorBoard, ModelCheckpoint, LearningRateScheduler
+import tensorflow as tf
+import matplotlib.pyplot as plt
+
+from learning_wavelets.config import LOGS_DIR, CHECKPOINTS_DIR
+from tensorflow.keras.optimizers import Adam
+from learning_wavelets.data.datasets import im_dataset_div2k, im_dataset_bsd500
+from learning_wavelets.models.exact_recon_unet import ExactReconUnet
+
+
+tf.random.set_seed(1)
+
+
+def train_unet(noise_std_train, noise_std_val, n_samples, source, cuda_visible_devices, base_n_filters):
+
+    os.environ["CUDA_VISIBLE_DEVICES"] = ','.join(cuda_visible_devices)
+    # data preparation
+    batch_size = 8
+    if source == 'bsd500':
+        data_func = im_dataset_bsd500
+    elif source == 'div2k':
+        data_func = im_dataset_div2k
+    im_ds_train = data_func(
+        mode='training',
+        batch_size=batch_size,
+        patch_size=256,
+        noise_std=noise_std_train,
+        return_noise_level=True,
+        n_samples=n_samples,
+    )
+    im_ds_val = data_func(
+        mode='validation',
+        batch_size=batch_size,
+        patch_size=256,
+        noise_std=noise_std_val,
+        return_noise_level=True,
+    )
+
+    n_epochs = 200
+    run_id = f'ExactReconUnet_{base_n_filters}_dynamic_st_{source}_{noise_std_train[0]}_{noise_std_train[1]}_{n_samples}_{int(time.time())}'
```

Comment:
```suggestion
    run_id = f'ExactReconUnet_{base_n_filters}_{source}_{noise_std_train[0]}_{noise_std_train[1]}_{n_samples}_{int(time.time())}'
```

We don't have a dynamic thresholding for the unet